SmartEVSE-3/src/evse.cpp

```diff
@@ -2635,7 +2635,7 @@ void ConfigureModbusMode(uint8_t newmode) {
             MBserver.registerWorker(BROADCAST_ADR, ANY_FUNCTION_CODE, &MBbroadcast);
 
             if (MainsMeter != EM_API) MBserver.registerWorker(MainsMeterAddress, ANY_FUNCTION_CODE, &MBMainsMeterResponse);
-            if (EVMeter) MBserver.registerWorker(EVMeterAddress, ANY_FUNCTION_CODE, &MBEVMeterResponse);
+            if (EVMeter != EM_API) MBserver.registerWorker(EVMeterAddress, ANY_FUNCTION_CODE, &MBEVMeterResponse);
```

Comment:
Should be: (EVMeter && EVMeter != EM_API)

Reply:
i agree.. the reason i diid is because the line above is if (MainsMeter != EM_API), should that not also be 
`if (MainsMeter && MainsMeter != EM_API)` then?

Reply:
You are absolutely right, I fixed that MainsMeter line already in master so we can keep it separated from your PR. Thx for noticing!